### PR TITLE
NN-5189: Get agencies for incident and hearing

### PR DESCRIPTION
**Pull request by natclamp-moj**
Uses the agency on the adjudication (originatingAgencyId) to get agency name and display on prisoner report.
Uses the agencyId listed on the hearing to get agency name to display on hearings page

### CODE REVIEW

File: server/services/reportedAdjudicationsService.ts [hunk 1]
```diff
@@ -592,7 +573,9 @@ export default class ReportedAdjudicationsService {
         return {
           hearing: {
             ...historyItem.hearing,
-            locationName: locationNamesByIdMap.get(historyItem.hearing.locationId),
+            locationName: `${locationNamesByIdMap.get(historyItem.hearing.locationId)}, ${agencyNameByLocationIdMap.get(
+              historyItem.hearing.agencyId
+            )}`,
```
Comment (neilmendum) [inline server/services/reportedAdjudicationsService.ts:578]: if tslint doesn't complain, could this line be
```
locationName: `${locationNamesByIdMap.get(historyItem.hearing.locationId)},
    ${agencyNameByLocationIdMap.get(historyItem.hearing.agencyId)}`,
```

File: server/services/reportedAdjudicationsService.ts [hunk 2]
```diff
@@ -479,7 +481,7 @@ export default class ReportedAdjudicationsService {
       },
       {
         label: 'Location',
-        value: `${location?.userDescription || ''}`,
+        value: `${location?.userDescription || ''}, ${agencyName.description}`,
```
Comment (neilmendum) [inline server/services/reportedAdjudicationsService.ts:484]: Do we have a test which covers `location` being null and so the `value` only equals `agencyName.description`?

File: server/services/reportedAdjudicationsService.ts [hunk 3]
```diff
@@ -584,6 +564,7 @@ export default class ReportedAdjudicationsService {
     if (!history.length) return []
     const hearings = history.filter((item: OutcomeDetailsHistory & HearingDetailsHistory) => !!item.hearing)
     const locationNamesByIdMap = await this.getHearingLocationMap(hearings, user)
+    const agencyNameByLocationIdMap = await this.getAgencyNameMap(hearings, user)
     const governorMap = await this.getAdjudicatorNameMap(hearings, user)
```
Comment (neilmendum) [inline server/services/reportedAdjudicationsService.ts:568]: Minor one - we could pass all of these into one promise, such as
```
    const [locationNamesByIdMap, agencyNameByLocationIdMap, governorMap] = await Promise.all([
      this.getHearingLocationMap(hearings, user),
      this.getAgencyNameMap(hearings, user),
      this.getAdjudicatorNameMap(hearings, user),
    ])
```

**Review by neilmendum — COMMENTED** [review]
Looks good - minor comments

File: server/services/reportedAdjudicationsService.ts (hunk 2 above)
Comment (natclamp-moj) [reply server/services/reportedAdjudicationsService.ts:484]: Embarrassingly, now that I check - there's no unit tests at all for this function 😱 I'll add some now - but yeah you're right, if location was null it's going to look a little odd with that comma and the agency

**Review by neilmendum — APPROVED** [review]
LGTM

**Pull request merged**